aws-lambda-java-events/src/main/java/com/amazonaws/services/lambda/runtime/events/CloudWatchAlarmEvent.java

```diff
@@ -0,0 +1,84 @@
+package com.amazonaws.services.lambda.runtime.events;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+import java.util.Map;
+
+@Data
+@Builder(setterPrefix = "with")
+@NoArgsConstructor
+@AllArgsConstructor
+public class CloudWatchAlarmEvent {
```

Comment:
let's add the new event here -> https://github.com/aws/aws-lambda-java-libs/blob/main/aws-lambda-java-events/README.md